fs/rails-base-graphql-api
# Add sign in activity

### Summary

This PR adds SignIn Activity

### How it works

![Screen Shot 2020-06-23 at 14 12 08](https://user-images.githubusercontent.com/58729845/85397074-971af880-b55b-11ea-9da1-736ce23d9f53.png)

### Test plan

List of steps to manually test introduced functionality:

* Open GraphiQL App
* Make request using schema:
```
mutation SignIn($email: String!, $password: String!) {
  signin(email: $email, password: $password) {
    me {
      id
      email
      activities {
        id
        title
        body
        event
        createdAt
      }
    }
    accessToken
    refreshToken
  }
}
```
* Check that response contents activity with event "USER LOGGED IN"

### Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduced DB fields have indexes and constraints
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### References
* Resolves #68
